### DISCUSSION TITLE
Do you know about vtubers? Ask Anything or Add your favorite!

### DISCUSSION THREAD

**Discussion by sergiodl-naist** [NAIST-SE/AttractingContributors#80] [General]
URL: https://github.com/sergiodl-naist/AttractingContributors-VTubers

Last and this year vtubers began to rise in popularity. I introduce the top 2020's YouTuber earner through SuperChats (donnations): Kiryu Coco!

Post a question about vtubers or recommend your favorite one!

**Comment by yokoro13** [comment]
I added!
Please help me!
https://github.com/yokoro13/AttractingContributors-VideoGames
